integration-tests/scenarios/stack_analyses_v2.md

```diff
@@ -1,28 +1,16 @@
-- Scenario: Check the analyzed dependencies for newer version of click package
-- Scenario: Check the analyzed dependencies for the exact version (arbitrary equality)
-- Scenario: Check the analyzed dependencies for older version of click package
-- Scenario: Check the analyzed dependencies for older version of click package with click>=5.0 in requirements
-- Scenario: Check the analyzed dependencies for older version of click package with click>5.0 in requirements
-- Scenario: Check the analyzed dependencies for older version of click package with click>=5.0, <=6.0 in requirements
-- Scenario: Check the analyzed dependencies for older version of click package with click>=5.0, <6.0 in requirements
-- Scenario: Check the analyzed dependencies for older version of click package with click>5.0, <6.0 in requirements
-- Scenario: Check the analyzed dependencies for older version of click package with click>5.0, <=6.0 in requirements
-- Scenario: Check the analyzed dependencies for click 6.7.*
-- Scenario: Check the integer normalization in requirements.txt for major and minor version numbers
-- Scenario: Check the analyzed dependencies part
-- Scenario: Check the outlier records
-- Scenario: Check that the API entry point requires authorization token
-- Scenario: Check that the API entry point requires authorization token
-- Scenario: Check that the stack analysis response for the pom.xml that contains only one component
-- Scenario: Check that the stack analysis response for the vertx.xml
-- Scenario: Check that the stack analysis response for the springboot.xml
+- Scenario: Check that the REST API point for the stack analyses accepts without user key
+- Scenario Outline: Check that the stack analysis REST API endpoint requires user key even for improper HTTP methods
+- Scenario Outline: Check that the stack analysis REST API endpoint does not accept any HTTP method other than POST
+- Scenario: Check that the REST API point for the stack analyses with external ID accepts without user key
+- Scenario Outline: Check that the stack analysis REST API endpoint requires user key even for improper HTTP methods
+- Scenario Outline: Check that the stack analysis REST API endpoint does not accept any HTTP method other than POST
+- Scenario: Check that the API entry point without user key
+- Scenario: Check that the API entry point with invalid user key
+- Scenario: Check that the API entry point with user key
+- Scenario Outline: Check the stack analysis v2 response when called with proper user key
+- Scenario Outline: Check the stack analysis v2 response for missing data when called with proper user key
+- Scenario Outline: Check the stack analysis v2 response for invalid manifest data when called with proper user key
+- Scenario Outline: Check the stack analysis v2 request and response when called with proper user key
+- Scenario Outline: Check the stack analysis v2 with known package data
+- Scenario: Check the outlier record
```

Comment:
can we add a test case which sends 100 requests in a miniute and overload the api then we should get a 429 status code

Reply:
This is not complete e2e test cases, these load test + snyk data verification test cases are still pending. As discussed in planning meeting we want to release test cases in phases.